zipkin-common/src/main/scala/com/twitter/zipkin/test/FunSuiteWithJUnit.scala

```diff
@@ -0,0 +1,9 @@
+package com.twitter.zipkin.test
+
+import org.scalatest.junit.JUnitRunner
+import org.junit.runner.RunWith
+import org.scalatest.FunSuite
+
+@RunWith(classOf[JUnitRunner])
+class FunSuiteWithJUnit extends FunSuite {
```

Comment:
Oh right.. This is a shared branch! OK I will help today but will be away
until Tuesday am Pacific. (FYI)